spring-cloud-starter-common-security-config-web/src/test/java/org/springframework/cloud/common/security/TestOAuthSecurityConfiguration.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016-2019 the original author or authors.
```

Comment:
The copyrights for all new files should be `2022` or (`2022-2022`). 

This comment serves as a "copyright check" for all other files (I will spare you the scattered ditto comments).